import.Dockerfile

```diff
@@ -1,23 +1,54 @@
-FROM quay.io/cdis/python:python3.9-buster-2.0.0
+ARG AZLINUX_BASE_VERSION=master
+
+# Base stage with python-build-base
+FROM quay.io/cdis/python-build-base:${AZLINUX_BASE_VERSION} AS base
 
 ENV appname=pelican
 
-ENV DEBIAN_FRONTEND=noninteractive
+# create gen3 user
+# Create a group 'gen3' with GID 1000 and a user 'gen3' with UID 1000
+RUN groupadd -g 1000 gen3 && \
+    useradd -m -s /bin/bash -u 1000 -g gen3 gen3
+
+# Install pipx
+RUN python3 -m pip install pipx && \
+    python3 -m pipx ensurepath
+
+USER gen3
+# Install Poetry via pipx
+RUN pipx install poetry
+ENV PATH="/home/gen3/.local/bin:${PATH}"
+USER root
+
+WORKDIR /${appname}
+
+# Builder stage
+FROM base AS builder
+
+RUN dnf update && dnf install -y \
+    python3-devel \
+    gcc \
+    postgresql-devel
+
+COPY . /${appname}
+
+# cache so that poetry install will run if these files change
+COPY poetry.lock pyproject.toml /${appname}/
 
-#RUN mkdir -p /usr/share/man/man1
-#RUN mkdir -p /usr/share/man/man7
+RUN poetry install -vv --no-interaction --without dev
```

Comment:
any reason we went from
```
RUN poetry config virtualenvs.create false \
    && poetry install -vv --no-dev --no-interaction \
    && poetry show -v
```
to this line?

Reply:
we are trying to create consistency between all of our services and use poetry to manage virtual environments instead of python. With the new AL base image PRs, we thought it would be a good time to do this!

Reply:
sounds good to me!